tests/acceptance/gcp-auth.bats

```diff
@@ -3,23 +3,25 @@
 #load _helpers
 #
 #SKIP_TEARDOWN=true
-VAULT_ADDR='http://127.0.0.1:8200'
-VAULT_IMAGE="${VAULT_IMAGE:-hashicorp/vault:1.9.0-rc1}"
+export VAULT_ADDR='http://127.0.0.1:8200'
+export VAULT_IMAGE="${VAULT_IMAGE:-hashicorp/vault:1.9.1}"
 
-if [[ -z SERVICE_ACCOUNT_ID ]]
+if [[ -z $SERVICE_ACCOUNT_ID ]]
 then
     echo "SERVICE_ACCOUNT_ID env is not set. Exiting.."
     exit 1
 fi
 
-if [[ -z PATH_TO_CREDS ]]
+if [[ -z $GOOGLE_APPLICATION_CREDENTIALS ]]
 then
-    echo "PATH_TO_CREDS env is not set. Exiting.."
+    echo "GOOGLE_APPLICATION_CREDENTIALS env is not set. Exiting.."
     exit 1
 fi
 
 export SETUP_TEARDOWN_OUTFILE=/tmp/output.log
 
+cp $GOOGLE_APPLICATION_CREDENTIALS ./creds.json
```

Comment:
should we clean this up (rm) after the test?

Alternatively, maybe we could use [$BATS_SUITE_TMPDIR](https://bats-core.readthedocs.io/en/stable/writing-tests.html#special-variables)

Reply:
Using `rm` in teardown